sonar/theme/templates/sonar/partial/dropdown_user.html

```diff
@@ -16,20 +16,27 @@
 #}
 
 <div class="dropdown-menu dropdown-menu-right" aria-labelledby="{{dropdownId}}">
-    <h6 class="dropdown-header">{{current_user_record.email}}</h6>
-    <a class="dropdown-item" href="{{ url_for('sonar.profile')}}">
-        {{_('Profile')}}
-    </a>
-    {% if has_superuser_access() %}
-    <a class="dropdown-item" href="{{ url_for('admin.index')}}">
-        {{_('Super administration')}}
-    </a>
-    {% endif %}
-    {% if has_admin_access() %}
-    <a class="dropdown-item" href="{{ url_for('sonar.manage')}}">
-        {{_('Administration')}}
-    </a>
-    {% endif %}
+  {% if current_user_record %}
```

Comment:
I think it's better to use the jinja block `{%-` to avoid spaces into output HTML. See "https://jinja.palletsprojects.com/en/2.11.x/templates/#whitespace-control"